pkg/provider/okta/okta_test.go

```diff
@@ -355,3 +356,191 @@ func TestGetStateToken(t *testing.T) {
 	assert.Nil(t, err)
 	assert.Equal(t, "token1", stateToken)
 }
+
+// anonymised from actual endpoint
+const fakeEndpointForm = `
+<form method="post" id="endpoint-health-form">
```

Comment:
I copied this out of dev tools and changed all the PII, so it's reasonably accurate